composer.json

```diff
@@ -85,6 +85,9 @@
             "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::clearCache",
             "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installAssets",
             "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installRequirementsFile"
+        ],
+        "build-parameters": [
```

Comment:
Why do you need that? `builParameters` is already called when you perform a `composer install` (or an update)